applications/conceptual-model-editor/src/action/utilities.ts

```diff
@@ -314,3 +327,30 @@ export function getRemovedAndAdded<T>(previousValues: T[], nextValues: T[]) {
     added
   };
 }
+
+/**
+ * @returns
```

Comment:
I can probably see the same information from the code. 
I would prefer to explain what is the semantic of this function.